Tasks/Kozinets/Task8/bl.py

```diff
@@ -0,0 +1,580 @@
+import data
+from os import system
+from time import sleep
+from decimal import Decimal
+from random import randint
+
+
+
+
+cart = [	]
+
+
+
+
+#######################################################################  start
+def represent_category():
+	"""takes nothing, returns list of names (keys) of items from choosen category"""
+
+	def category_data_keys(category:str):
+		for dict in data.db:
+			for key in dict:
+				if dict[key][0] == category:
+					return sorted(dict)
```

Comment:
why do you need this function defined inside another one, if you are not going to use closure??